### DISCUSSION TITLE
SIDESTORE NOT VERIFIED AFTER ONE MONTH

### DISCUSSION THREAD

**Discussion by bishoysiu** [SideStore/SideStore#651] [Q&A] [answered]
Can anyone help me, I side loaded sidestore and it was working fine, after one day; it crashes and can’t be accessed. I thought it was a glitch so i re-sideloaded sidestore and after a day it stopped working. I checked and found out that Sidestore wasn't verified anymore. The other apps that I downloaded through sidestore are verified though. Does anyone know the fix?
![image](https://github.com/SideStore/SideStore/assets/168586080/6e4ebefc-40a5-417f-88e2-d14eb33ddec3)

**Comment by Spidy123222** [comment, marked as answer]
This thread is for discussions not issues but you can fix this by refreshing SideStore right after installing it to prevent this.
